server/builtin.go

```diff
@@ -22,7 +22,7 @@ type ConsensusType string
 const (
 	DevConsensus     ConsensusType = "dev"
 	IBFTConsensus    ConsensusType = "ibft"
-	PolyBFTConsensus ConsensusType = "polybft"
+	PolyBFTConsensus ConsensusType = consensusPolyBFT.PolyBFTConsensusName
```

Comment:
This doesn't seem consistent with the other constants in this group

Reply:
Fixed https://github.com/0xPolygon/polygon-edge/pull/1005/commits/f432f4babd18c4081d8bd25c8ade9a73badfbc28